jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/content/ContentSourceCompletableFuture.java

```diff
@@ -53,25 +66,34 @@
  * String s = cs.get();
  * }</pre>
  */
-public abstract class ContentSourceCompletableFuture<X> extends CompletableFuture<X>
+public abstract class ContentSourceCompletableFuture<X> extends CompletableFuture<X> implements Runnable, Invocable
```

Comment:
Do not implement `Runnable` and `Invocable` but solely `Invocable.Task` instead.